cli_reference/developer_cli_odo/odo-migration-guide.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id='odo-migration-guide']
+= Migrating from odo v2 to v3
+include::_attributes/common-attributes.adoc[]
+:context: odo-migration-guide
+
+toc::[]
+
+`odo` 3.0 includes many new commands to improve the developer experience, providing the same functionality as before, but in a faster and more efficient manner.
```

Comment:
Minimalism. What do you think of this rephrase?

``odo` 3.0 includes many new commands to improve the developer experience, providing faster and more efficient functionality.`

Reply:
I think I might leave it, as this was written upstream and I don't want to diverge the content too much